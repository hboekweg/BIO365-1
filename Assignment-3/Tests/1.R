source("1.R")

print(sortFactor(c("hot", "warm", "cold", "hot", "warm", "warm")))
print(sortFactor(c("1", "One", "Won", "Wun", "Oun")))
print(sortFactor(c("cat", "dog", "hamster", "rabbit", "snake", "dog")))