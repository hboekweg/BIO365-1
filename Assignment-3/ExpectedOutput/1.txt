[1] hot  warm cold hot  warm warm
Levels: cold hot warm
[1] 1   One Won Wun Oun
Levels: 1 One Oun Won Wun
[1] cat     dog     hamster rabbit  snake   dog    
Levels: cat dog hamster rabbit snake
